gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/Oplog.java

```diff
@@ -772,30 +819,33 @@
    *          The previous oplog
    */
   private Oplog(long oplogId, DirectoryHolder dirHolder, Oplog prevOplog) {
+    this(oplogId, prevOplog.oplogSet, dirHolder, prevOplog.getDataVersionIfOld());
+  }
+
+
+  // Asif: StandBy oplog to be used in case of disk full, to allow
+  //  rollback logging etc.
+  // When this oplog is used, it is assumed that before the switch to
+  // this oplog, the compactor thread will be stopped. This means that
+  // it is guaranteed that previous oplog will not be compacted
+  //( as the compactor is stopped while the previous oplog is not yet
+  // added for compaction. This means that fixed data like RVV etc
+  //  need not be added to standby oplog (atleast that is my understanding))
```

Comment:
Even I am not sure. Pulling in Suranjan also for review.
@suranjan , @sumwale ?

Reply:
I am assuming we are ok with this...?

Reply:
With KRF being allowed to be formed with more disk space. Right?

Reply:
yes